docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -42,18 +42,67 @@ components:
 
 ## Estimating transaction costs
 
-:::warning Method deactivated
+Use the [`linea_estimateGas`](/developers/reference/api/linea-estimategas) API method to estimate the 
+gas cost for sending a transaction. The method returns the recommended gas limit, the base fee per 
+gas, and the priority fee per gas.
 
-The `linea_estimateGas` method used to estimate transaction fees has been temporarily deactivated
-on Linea Mainnet and testnets.
+:::info important
 
+Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
+[`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
+[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory), but 
+they usually return a higher (and less accurate) estimate than `linea_estimateGas`. 
+
+We recommend you use `linea_estimateGas` where possible to maximize the accuracy of the gas 
+estimate.
+
+:::
+
+The following example executes `linea_estimateGas` on the Linea mainnet using the public RPC URL.
+
+:::tip
+
+You can also call the API using [Infura's supported Linea endpoints](https://docs.infura.io/api/networks/linea/choose-a-network).
 :::
 
-As Linea supports all but a few [Ethereum JSON-RPC API methods](developers/guides/linea-api), there are several 
-options for estimating gas. Click through for Infura's guides on their usage:
-- [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas) — 
-Generates and returns an estimate of how much gas is necessary to allow the transaction to complete.
-- [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice) — Returns 
-the current gas price in wei.
-- [`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory) — 
-Returns historical gas information.
+<Tabs className="my-tabs">
+  <TabItem value="curl" label="curl" default>
+
+```bash
+curl -X POST --data '{"jsonrpc": "2.0","method": "linea_estimateGas","params": [{"from": "0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73","gasPrice":"0x100000000","gas":"0x21000"}],"id": 53}' https://rpc.linea.build
```

Comment:
This won't work until it's activated on mainnet, so should we use the sepolia endpoint?